### PR TITLE
Add Spotify icon

### PR DESCRIPTION
**Pull request by ShimiSun**
thank you @JetLim for your work here!

### CODE REVIEW

**Comment by ShimiSun** [comment]
The reason this PR is also making changes on the Instagram files is because I was a rookie in the past and edited the Instagram files manually when the icon was changed, instead of running `yarn generate-icons`. 
The current file changes are derived from the SVG icon I got from Chris using `yarn generate-icons`.

File: src/js/icons/Instagram.js [hunk 1]
```diff
@@ -4,6 +4,6 @@ import { StyledIcon } from '../StyledIcon';
...
....1621583,15.2041299 2.1621583,12.0000238 C2.1621583,8.79587013 2.17440026,8.41632182 2.23213276,7.15097025 C2.28548292,5.9809821 2.48097316,5.34559117 2.64531072,4.92274348 C2.86299845,4.3626144 3.12303286,3.96286924 3.54297487,3.54297487 C3.96286924,3.12303286 4.3626144,2.86299845 4.92274348,2.64531072 C5.34559117,2.48097316 5.9809821,2.28548292 7.15097025,2.23213276 C8.41632182,2.17440026 8.79587013,2.1621583 12.0000238,2.1621583 Z M12.0000238,16 C9.79085069,16 8,14.2091493 8,12.0000238 C8,9.79085069 9.79085069,8 12.0000238,8 C14.2091493,8 16,9.79085069 16,12.0000238 C16,14.2091493 14.2091493,16 12.0000238,16 Z M12.0000238,5.8378417 C8.59671206,5.8378417 5.8378417,8.59671206 5.8378417,12.0000238 C5.8378417,15.4032879 8.59671206,18.1621583 12.0000238,18.1621583 C15.4032879,18.1621583 18.1621583,15.4032879 18.1621583,12.0000238 C18.1621583,8.59671206 15.4032879,5.8378417 12.0000238,5.8378417 Z M19.8456418,5.59438394 C19.8456418,6.38968246 19.2009146,7.03436203 18.4056161,7.03436203 C17.6103652,7.03436203 16.965638,6.38968246 16.965638,5.59438394 C16.965638,4.79908543 17.6103652,4.15435822 18.4056161,4.15435822 C19.2009146,4.15435822 19.8456418,4.79908543 19.8456418,5.59438394 Z' />
```
Comment (ericsoderberghp) [inline src/js/icons/Instagram.js]: Why is the Instagram fill a url?

Comment (ShimiSun) [reply src/js/icons/Instagram.js]: Check out my comment above

Comment (ericsoderberghp) [reply src/js/icons/Instagram.js]: I don't think that will work. Take a look at instagram.svg. "instagram-a" is an internal reference to a linear gradient that is being stripped when `generate-icons` is run.

Comment (ShimiSun) [reply src/js/icons/Instagram.js]: well, I agree it looks strange, but if you do a fresh clone run `yarn install && yarn generate-icons` this file will be updated. doesn't mean we need to change something on the svg?

Comment (ShimiSun) [reply src/js/icons/Instagram.js]: I removed the Instagram changes to eliminate the bottleneck of this PR. we'll discuss it offline. @ericsoderberghp

**Pull request merged**